src/components/registerDelegate/index.test.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { expect } from 'chai';
+import { mount } from 'enzyme';
+import { BrowserRouter as Router } from 'react-router-dom';
+import RegisterDelegate from './registerDelegate';
+import RegisterDelegateConnected from './index';
+import { accountUpdated } from '../../actions/account';
+
+describe('RegisterDelegateConnected', () => {
+  let mountedAccount;
+  // Mocking store
+  const peers = {
+    status: {
+      online: false,
+    },
+    data: {
+      currentPeer: 'localhost',
+      port: 4000,
+      options: {
+        name: 'Custom Node',
+      },
+    },
+  };
+
+  const account = {
+    isDelegate: false,
+    address: '16313739661670634666L',
+    username: 'lisk-nano',
+  };
+
+  const store = {
+    dispatch: () => {},
+    subscribe: () => {},
```

Comment:
You can use `redux-mock-store` like here:
https://github.com/LiskHQ/lisk-nano/blob/development/src/components/app/index.test.js#L6